src/Resources/translations/SonataPageBundle.bg.xliff

```diff
@@ -562,8 +562,8 @@
         <source>list.label_edited</source>
         <target>Редактирана</target>
       </trans-unit>
-      <trans-unit id="141" resname="show.label_site">
-        <source>show.label_site</source>
+      <trans-unit id="141" resname="show.group_site">
```

Comment:
When it's used ? I dont see the related change in the code

Reply:
Show site page: in `http://localhost:8001/admin/app/sonatapagesite/1/show`
I guess generator of translation keys changed from `show.label_site` to `show.group_site`

Reply:
Weird...

But if it works better this way, ok ! Thanks

Reply:
Page :
![Screenshot 2022-07-28 at 16 56 29](https://user-images.githubusercontent.com/841946/181571082-b2472ce9-5b2c-43c1-8b9d-188635e9ef7f.png)

Translation :
![Screenshot 2022-07-28 at 16 55 10](https://user-images.githubusercontent.com/841946/181570938-b4db490e-9e14-4b18-89c8-65b6d2fc0993.png)